pkgs/tools/misc/arxiv-latex-cleaner/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, python3Packages
+, fetchFromGitHub
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "google-research";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = pname;
```

These shouldn't share a variable, since they're semantically different.
(https://github.com/NixOS/nixpkgs/pull/260020#discussion_r1350727748)

```suggestion
    owner = "google-research";
```